modules/openy_features/openy_location/modules/openy_loc_filter/openy_loc_filter.install

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * @file
+ * Open Y Location filter install file.
+ */
+
+use Drupal\node\NodeInterface;
+
+/**
+ * Set default locations.
+ */
+function openy_loc_filter_install() {
+  $config = \Drupal::config('openy_loc_filter.location_filter_settings')->get('locations');
+
+  $branches_list = [
+    'branch' => [],
+    'camp' => [],
+  ];
+
+  /** @var \Drupal\Core\Database\Query\SelectInterface $query */
+  $query = \Drupal::database()->select('node_field_data', 'n')
+    ->fields('n', ['nid', 'title', 'type'])
+    ->condition('type', ['branch', 'camp'], 'IN')
+    ->condition('status', NodeInterface::PUBLISHED);
+  $items = $query->execute()->fetchAll();
+  foreach ($items as $item) {
+    $branches_list[$item->type][$item->nid] = $item->title;
+  }
+
+  $branches_list = $branches_list['branch'] + $branches_list['camp'];
```

Comment:
@AleksandrPrykhodkoDev What is the reason to sort items by type in a loop if then you merge them into one array here?